libbpf-rs/Cargo.toml

```diff
@@ -38,6 +38,7 @@ probe = "0.3"
 scopeguard = "1.1"
 serial_test = "0.5"
 tempfile = "3.3"
+memmem = "0.1.1"
```

Comment:
Can you please keep the list sorted? Thanks!

Reply:
Done